c/src/ml-api-common.c

```diff
@@ -944,21 +979,29 @@ ml_tensors_data_create (const ml_tensors_info_h info, ml_tensors_data_h * data)
         status);
   }
 
+  _info = (ml_tensors_info_s *) info;
+  if (_info->info.format == _NNS_TENSOR_FORMAT_FLEXIBLE) {
+    _ml_logw
+        ("[ml_tensors_data_create] format is FLEXIBLE, skipping tensor memory allocation. "
+        "In this case, tensor_data->tensors[i].data will be allocated in ml_tensors_data_set_tensor_data() with the data size.");
```

Comment:
let's avoid printing data structure (tensor_data->tensors[i].data) in log message.
```
[ml_tensors_data_create] format is FLEXIBLE, skipping tensor memory allocation.
Use ml_tensors_data_set_tensor_data() to update data buffer.
```

Reply:
Thank you. I've updated it.